packages/trpc/server/routers/viewer/workflows/getAllActiveWorkflows.handler.ts

```diff
@@ -0,0 +1,19 @@
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import type { TGetAllActiveWorkflowsInputSchema } from "./getAllActiveWorkflows.schema";
+import { getAllWorkflowsFromEventType } from "./util";
+
+type GetAllActiveWorkflowsOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetAllActiveWorkflowsInputSchema;
+};
+
+export const getAllActiveWorkflowsHandler = async ({ input }: GetAllActiveWorkflowsOptions) => {
+  const { eventType } = input;
```

Comment:
![image](https://github.com/calcom/cal.com/assets/1780212/e736d6b9-bba9-40ef-8cf8-e78a0d9130a7)
This doesn't seem to have any authorization check and I am able to retrieve different org's workflows

Reply:
Thank you for catching that 🙏 added an authorization check now